### DISCUSSION TITLE
is border-opacity no longer supported in tailwind 3?

### DISCUSSION THREAD

**Discussion by oshi97** [tailwindlabs/tailwindcss#7320] [Help] [answered]
Couldn't see anything about it in the upgrade guide, but it's also missing from the v3 docs. Locally it doesn't seem to do anything anymore?

**Comment by ExEr7um** [comment, marked as answer]
It's not missing but now it uses different syntax. You should use it like this:
```html
<div class="border-4 border-indigo-500/100 ..."></div>
<div class="border-4 border-indigo-500/75 ..."></div>
<div class="border-4 border-indigo-500/50 ..."></div>
```
You can see it [here](https://tailwindcss.com/docs/border-color) in docs under the "Changing the opacity" heading.

**Reply by oshi97** [reply]
thanks!
